public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>React Data App</title>
```

Comment:
```suggestion
    <title>React Data App</title>
```
    <title>React Sample App</title>